std/os/linux/test.zig

```diff
@@ -83,3 +83,12 @@ test "dl_iterate_phdr" {
     expect(linux.dl_iterate_phdr(usize, iter_fn, &counter) != 0);
     expect(counter != 0);
 }
+
+test "sigaltstack" {
+    var st: linux.stack_t = undefined;
```

Comment:
can you add the wrapper function in std.os? Then this test can go into std/os/test.zig and test all the OS's including libc-linked builds.

Reply:
> can you add the wrapper function in std.os

I thought that was only for abstractions that were valid across all the OSs.

Reply:
it's a candidate for os.zig if:
 * it's available on more than 1 os (or available on libc and non libc for linux), or
 * it would be worthwhile to have zig-friendly function signature